starter-code/package.json

```diff
@@ -6,12 +6,20 @@
     "start": "node ./bin/www"
```

Comment:
You are missing the configuration for being able to debug from Visual Studio Code (launch.json).